src/EnergyPlus/ZoneEquipmentManager.cc

```diff
@@ -1322,6 +1322,34 @@ void SetUpZoneSizingArrays(EnergyPlusData &state)
                                      state.dataSize->CalcFinalZoneSizing(CtrlZoneNum).MinOA);
         }
     }
+
+    // Populate DesignSpecification:OutdoorAir:SpaceList spaces
```

Comment:
Move this up so it's done for all `OARequirements`, not just the ones referenced in a Sizing:Zone object, because `OARequirements` can be used in other places, too, like Controller:MechanicalVentilation and air terminal units.